interfaces/fiveg_n2/v0/schema.py

```diff
@@ -0,0 +1,32 @@
+"""This file defines the schemas for the provider and requirer sides of the `fiveg_n2` interface.
+It exposes two interfaces.schema_base.DataBagSchema subclasses called:
+- ProviderSchema
+- RequirerSchema
+Examples:
+    ProviderSchema:
+        unit: <empty>
+        app: {
+            "amf_hostname": "amf",
+            "ngapp_port": 38412
```

Comment:
Why use `amf_hostname` and `ngapp_port`. It's somewhat confusing as we're talking about the same object.

`amf_hostname` and `amf_port` would make more sense in my opinion and you may want to specify somewhere that the protocol of the n2 interface is `NGAP`

Reply:
Done.